developments_build/02_build_devdb.sh

```diff
@@ -12,7 +12,7 @@ docker run --rm\
     -v $(pwd):/src\
     -w /src\
     -e BUILD_ENGINE=$BUILD_ENGINE\
-    nycplanning/docker-geosupport:latest bash -c "
+    nycplanning/docker-geosupport:20.3.0 bash -c "
```

Comment:
let's pass it from https://github.com/NYCPlanning/db-developments/blob/dev/developments_build/version.env
some thing like `VERSION_GEO=20.3.0`